source/codegen/metadata/nidigital/functions.py

```diff
@@ -3077,6 +3077,93 @@
                 "use_array": True
             }
         ],
-        "returns": "ViStatus"
+        'returns': 'ViStatus'
+    },
+    'ReadStatic': {
```

Comment:
Since you are not handling ReadStatic in this PR anyway, can you just remove this metadata and add it whenever support for  ViUInt8[] is added?

Reply:
ReadStatic can be removed since it uses ivi-dance-with-a-twist. But arrays need to be handled in this PR itself properly because the current way of using repeated uint32 is wrong and PR cannot to owners like that. @doshirohan do you agree?

Reply:
Sure, let's handle arrays as well here, but that also means you will have to write unit test for `ViUInt8[]` using nifake, either in this PR or a follow up PR. I would prefer this PR since it shouldn't be much of an effort. If you are planning to write test in follow up PR, call that out in the description.

Reply:
I will have to make slight changes in mako files to handle ViUInt8[], so I will do that in separate PR.

Reply:
@ni-pkhare , Rohan meant that ni fake tests can be either this PR or follow up PR. But we are telling you to not send the current way of using repeated uint32 as it is for owners because that wont work and wont be accepted by owners

Reply:
@ni-pkhare  If the mako file changes affect the auto generation, that needs to be handled in this PR only. Since we are not confident enough, I would suggest adding support for ViUInt8 and ViUInt8[] in a separate PR targeted to feature branch where you make codegen changes add fake metadata and test for that. Only after that gets approved and submitted, you can come back to this current task of adding meta data for the Digital Pattern APIs.